christfellowshipchurch/web-app-v2
# Location Search

### About
This PR implements the new location changes we want to implement. User can now input an address and the cards will be reordered based on distance from the user address.

### Test Instructions
1. Go to `/locations` on the testing link and test to ensure that when searching for an address the locations display as expected.
2. Ensure that the UI looks like our Figma design.
3. Ensure that the loader works as expected.
4. Ensure that the page scrolls down to results when using user's location or clicking the search button.

### Screenshots
<img width="1440" alt="Screenshot 2023-11-29 at 1 52 47 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/f1513904-cea9-4d11-b8b5-e3d6ed02701f">


### Closes Tickets
[CFDP-2776](https://christfellowshipchurch.atlassian.net/browse/CFDP-2776)



[CFDP-2776]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
